app/assets/javascripts/filter.js

```diff
@@ -0,0 +1,82 @@
+$("#filter_projects").on("click", function() {
+      $("#filters").animate({width: "toggle"}, 1000);
+    });
+
+$(document).ready(function() {
+  var auth_token = "<%= form_authenticity_token %>"
+
+  var cleanFilters = function(filters) {
+    var cleanedFilters = [];
+    filters.forEach(function(element) {
+      cleanedFilters.push(element.match(/\[(.*?)\]/)[1]);
+    });
+    return cleanedFilters;
+  }
+
+  var appendProject = function(project) {
+    $("#projects").append(
+      "<div class='col-md-3'><a href='/" + project.organization + "/projects/" + project.id + "'>" + project.title + "</a>" +
+      "<a href='/" + project.organization + "/projects/" + project.id + "'><img src='" + project.image_url +
+      "' width='250px' height='auto'></a>" + "<div class='project-desc'>" + project.description + "</div>" +
+      '<form action="/pending_loan" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden"' +
+      'value="&#x2713;" /><input type="hidden" name="authenticity_token" value="' + auth_token + '" />' +
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Identifier 'auth_token' is not in camel case.